docs/guides/4-custom-rulesets.md

```diff
@@ -19,41 +19,58 @@ rules:
 
 Spectral has [built-in functions](../reference/functions.md) such as `truthy` or `pattern`, which can be used to power rules.
 
-<!-- theme: info -->
+### Given
 
-> Spectral processes each rule on "resolved document" (a file where all `$ref` entries have been resolved). If your rule needs to access `$ref` values, you must set `resolved: false` on your rule.
+The `given` property is one of only two on required properties on each rule definition (the other being `then`).
+
+It can be any valid JSONPath expression or an array of JSONPath expressions.
+[JSONPath Online Evaluator](http://jsonpath.com/) is a helpful tool to determine what `given` path you want.
+
+### Severity
+
+The `severity` keyword is optional and can be `error`, `warn`, `info`, or `hint`.
+
+The default value is `warn`.
+
+### Resolved
+
+By default, Spectral processes each rule on a "resolved" document (a file where
+all `$ref` JSON schema references have been replaced with the objects they point
```

Comment:
```suggestion
all `$ref` JSON Schema references have been replaced with the objects they point
```

Sorry to not but we gotta do this or people complain. There are a few schema languages in JSON but only one JSON Schema.